### ISSUE TITLE
How is the fuzzing run / fuzz testing as part of CI

### ISSUE THREAD

**Issue by dee-see** [kivikakk/comrak#353]
Hello :wave: 

I was happy to see that `cargo fuzz` is configured for this project! I was wondering what's the workflow to run fuzz testing in your development cycle? Is it ad hoc or is there a scheduled job configured somewhere?

If it's not scheduled, would you be open to a contribution that adds a CI job that runs on a schedule or would that exhaust GitHub Action free minutes? I'm not sure how GitHub Action minutes work.

Thank you!

**Comment by rlidwka** [comment]
I don't think fuzzing should be run on CI ever.

CI tasks should be:
1. quick (so any pull request gets immediate feedback)
2. reproduceable (so if something fails, you know it failed because of last commit)

And neither of those are what fuzzing is known for.

**Comment by kivikakk** [comment]
Indeed, it is ad hoc. I am inclined to agree with @rlidwka's comment; it'd also take some effort to preserve the fuzzing run information between runs (so that we continue to explore new paths).

**Comment by digitalmoksha** [comment]
Yeah, that makes sense
